src/regions/de/i18n/steps.de.ts

```diff
@@ -2,8 +2,9 @@ export default {
   intro1: 'Hallo!',
   intro2:
     'Ich bin Chloe, eine automatische medizinische Assistentin. Ich bin hier, um Ihnen genaue Informationen über COVID-19 zu geben. Um sie für Ihre Situation relevant zu machen, möchte ich Ihnen ein paar Fragen stellen. Es sollte nicht länger als ein paar Minuten dauern.',
-  consent:
-    'Sind Sie damit einverstanden, in [Übereinstimmung mit den Dienstbedingungen](https://github.com/dialoguemd/covid-19/wiki/German-Privacy-Page) und [der Datenschutzpolitik vorzugehen](https://github.com/dialoguemd/covid-19/wiki/German-Data-Protection-page)?',
+  consent: `Zur Verarbeitung Ihrer personenbezogenen Daten benötige ich Ihre Einwilligung. Wir verarbeiten Ihre Daten lediglich, um für Sie relevante Informationen zu COVID-19 zur Verfügung zu stellen und diese Anwendung weiter zu verbessern. Die Einwilligung kann jederzeit ohne Angabe von Gründen widerrufen werden. Alle Informationen zum Widerruf und zu den Einzelheiten der Datenverarbeitung finden Sie in unserer [Datenschutzerklärung](https://github.com/dialoguemd/covid-19/wiki/Datenschutzerkl%C3%A4rung). 
+    Mit der Verarbeitung Ihrer personenbezogenen Daten (einschließlich Gesundheitsdaten) durch die Argumed Consulting Group GmbH, Ohmstraße 11, 80802 München, und die Dialogue Technologies Inc., 390 Notre-Dame St W #200 Montreal, QC H2Y 1T9 sind Sie einverstanden?
+    `,
```

Comment:
did you check it rendered properly? for paragraphs, you need a full blank line between paragraphs, it's possible you need to start text on a newline instead of next to backtick, as indentation is determined relative to first line of text.

Reply:
i.e. 
```ts
    consent: `
      Zur Verarbeitung Ihrer personenbezogenen Daten benötige ich Ihre Einwilligung. Wir verarbeiten Ihre Daten lediglich, um für Sie relevante Informationen zu COVID-19 zur Verfügung zu stellen und diese Anwendung weiter zu verbessern. Die Einwilligung kann jederzeit ohne Angabe von Gründen widerrufen werden. Alle Informationen zum Widerruf und zu den Einzelheiten der Datenverarbeitung finden Sie in unserer [Datenschutzerklärung](https://github.com/dialoguemd/covid-19/wiki/Datenschutzerkl%C3%A4rung).

      Mit der Verarbeitung Ihrer personenbezogenen Daten (einschließlich Gesundheitsdaten) durch die Argumed Consulting Group GmbH, Ohmstraße 11, 80802 München, und die Dialogue Technologies Inc., 390 Notre-Dame St W #200 Montreal, QC H2Y 1T9 sind Sie einverstanden?
`
```